torch/_library/utils.py

```diff
@@ -158,6 +158,24 @@ def mutates_and_returns_first_arg(op: torch._ops.OpOverload):
     return True
 
 
+def fill_defaults(schema, args, kwargs):
```

Comment:
We don't already have a version of that somewhere? :o 
I'm sure this is something that should be in our subclass utils :p

Reply:
I have not been able to find one. There's something similar for torch_function and ONNX but those are not exactly what we want